### DISCUSSION TITLE
Listado de Historias de usuarios

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [NataliaAlvarezIspc/proyecto-ispc-ecommerce#31] [General]
Creo esta discusión  para ir agregando las historias de usuarios que se nos vayan ocurriendo y las tareas que corresponden a esas historias de usuarios. Por favor escribanlas con la redacción correcta para historias de usuarios. Recuerden ir siguiendo un hilo dentro del manejo de la página, ej: no puedo pagar si antes no elegí que comprar.
Poner atención a los numero de historias ya creadas

**Comment by NataliaAlvarezIspc** [comment]
https://github.com/NataliaAlvarezIspc/proyecto-ispc-ecommerce/issues/29 #US07 como usuario cliente quiero tener acceso al catálogo de gustos

https://github.com/NataliaAlvarezIspc/proyecto-ispc-ecommerce/issues/35 #US12 Como usuario cliente quiero poder elegir cantidades

Acá dos historias de usuario que cree

**Reply by rpgrca** [reply]
Estoy suponiendo que estamos simplificando al máximo, somos heladería artesanal y vendemos 1/4, 1/2 y 1kg nomás en potes de tergopol, no vendemos ni bombones helados ni alfajores ni palito ni nada de eso, no?

**Reply by NataliaAlvarezIspc** [reply]
si, en la historia dice cuarto medio o kilo

**Comment by BraianTroncoso** [comment]
Como usuario quiero saber que gustos estan disponibles

**Reply by NataliaAlvarezIspc** [reply]
Esta ya está, es la primera en el comentario de más arriba.

**Comment by BraianTroncoso** [comment]
Como usuario quiero saber que gustos tienen azúcar y cual no.

**Comment by Brian-Almada** [comment]
Como usuario quiero saber la cantidad límite de gustos que tiene cada presentación

**Comment by KAAcevedo** [comment]
En estos días estuve pensando varias User Stories, aclaro que me guié en las cuatro volcadas en la Wiki:

#US05: Como usuario cliente, cuando quiero buscar un producto específico puedo utilizar "la lupita" de la barra de navegación para encontrarlo fácilmente.

#US06: Como usuario cliente, cuando quiero ver la información detallada de un producto puedo hacer clic en su imagen o nombre para acceder a su página individual.

#US07: Como usuario cliente, cuando quiero agregar un producto a mi carrito de compras puedo hacer clic en el botón "Agregar al carrito" en la página del producto.

#US08: Como usuario cliente, cuando quiero ver los productos que tengo en mi carrito puedo hacer clic en el ícono del carrito en la esquina superior derecha de la pantalla.

#US09: Como usuario cliente, cuando quiero eliminar un producto de mi carrito puedo hacer clic en el botón "Eliminar" al lado del producto correspondiente en la página del carrito.

#US10: Como usuario cliente, cuando quiero realizar el pago de mi carrito de compras puedo hacer clic en el botón "Pagar" para ingresar mis datos de pago y completar la transacción.

**Comment by NataliaAlvarezIspc** [comment]
Así quedarían las historias de usuarios. Todavía hay que hacer correcciones en las issue por eso no están linkeadas. Comenten si les parece que están bien, así ya las paso a la documentación.


•	[#US01] Como usuario cliente nuevo cuando quiero usar el sitio entonces tengo que registrarme.
•	[#US02] Como usuario cliente existente cuando quiero usar el sitio entonces tengo que loguearme con usuario y contraseña.
•	[#US03] Como usuario administrador del sitio cuando quiero crear nuevos tipos de producto entonces utilizo la página de tipos de producto.
•	[#US04] Como usuario administrador cuando quiero crear un nuevo producto entonces voy a la página de ABM de productos.
•	[#US05] Como usuario cliente, si olvido mi contraseña quiero poder recuperarla
•	[#US06] Como usuario cliente quiero tener la posibilidad de modificar datos de registración. 
•	[#US07] Como usuario cliente quiero tener acceso al catálogo. 
•	[#US08] Como usuario cliente, una vez en el catálogo quiero poder seleccionar entre el listado de productos a comprar. 
•	[#US09] Como usuario cliente quiero poder elegir la cantidad de artículos que voy a comprar.
•	[#US10] Como usuario cliente, cuando quiero buscar un producto específico puedo utilizar "la lupita" de la barra de navegación para encontrarlo fácilmente.
•	[#US11] Como usuario cliente, cuando quiero ver la información detallada de un producto puedo hacer clic en su imagen o nombre para acceder a su página individual.
•	[#US12] Como usuario cliente, cuando quiero ver los productos que tengo en mi carrito puedo hacer click en el ícono del carrito en la esquina superior derecha de la pantalla.
•	[#US13] Como usuario cliente, al finalizar compra, deseo ver la vista previa del carrito. 
•	[#US14]  Como usuario cliente deseo poder modificar o eliminar los productos del carrito antes de finalizar la compra. 
•	[#US15] Como usuario cliente, cuando quiero realizar el pago de mi carrito de compras puedo hacer clic en el botón "Pagar" para ingresar mis datos de pago y completar la transacción.
•	[#US16] Como usuario cliente deseo poder seleccionar la forma de pago. 
•	[#US17] Como usuario cliente  quiero poder seleccionar entre retirar mi pedio en la tienda o solicitar delivery.

•	[#US18] Como usuario cliente, una vez que finalice todos los pasos de la compra, quiero ver una notificación confirmando que la misma fue exitosa.
•	[#US19] Como usuario cliente si quiero tomar contacto con el adminstrador debo completar el formulario de contacto
•	[#US20] Como usuario administrador deseo tener acceso al listado de ventas.
•	[#US21] Como usuario administrador quiero poder acceder al stock de productos.

**Reply by BraianTroncoso** [reply]
Para mi estan bien, agregaria solamente que como Usuario cliente quisiera ver primero que todo que ofertas hay.

**Comment by rpgrca** [comment]
También me parece bien. Yo mantendría lo de las ofertas al mínimo posible porque con una persona menos no sé si podremos agregar algo como se lo pensó en su momento.

**Reply by BraianTroncoso** [reply]
Si, tenes razón, concuerdo.
